LEAF_Request_Portal/utils/LEAF_importStandardConfig.php

```diff
@@ -100,7 +100,49 @@ function importTable($db, $tempFolder, $table) {
 importTable($db, $tempFolder, 'workflow_routes');
 importTable($db, $tempFolder, 'step_modules');
 
+if (is_dir($paste_custom_templates) && is_dir($copy_custom_templates)) {
+    // remove files from templates/email/custom_override on subordinate
+    $files = glob($paste_custom_templates . '*');
+
+    // Deleting all the files in the list
+    foreach ($files as $file) {
+        if(is_file($file)) {
+            // Delete the given file
+            unlink($file);
+        }
+    }
+
+    // copy templates/email/custom_override from primary to subordinate
+    copyDirectory($copy_custom_templates, $paste_custom_templates);
+}
+
 $db->query("ALTER TABLE `records_dependencies` ADD CONSTRAINT `fk_records_dependencyID` FOREIGN KEY (`dependencyID`) REFERENCES `dependencies`(`dependencyID`) ON DELETE RESTRICT ON UPDATE RESTRICT;");
 $db->query("ALTER TABLE `dependency_privs` ADD CONSTRAINT `fk_privs_dependencyID` FOREIGN KEY (`dependencyID`) REFERENCES `dependencies`(`dependencyID`) ON DELETE RESTRICT ON UPDATE RESTRICT;");
 $db->query("ALTER TABLE `category_count` ADD CONSTRAINT `category_count_ibfk_1` FOREIGN KEY (`categoryID`) REFERENCES `categories`(`categoryID`) ON DELETE RESTRICT ON UPDATE RESTRICT;");
 $db->query("ALTER TABLE `category_privs` ADD CONSTRAINT `category_privs_ibfk_2` FOREIGN KEY (`categoryID`) REFERENCES `categories`(`categoryID`) ON DELETE RESTRICT ON UPDATE RESTRICT;");
+
+function copyDirectory($source, $destination) {
+    $files = scandir($source);
+
+    foreach ($files as $file) {
+        if ($file !== '.' && $file !== '..') {
+            // not sure if the / is needed here

```

Comment:
Since the code is intended to be used in prod, this should be checked so we're sure.

The first iteration starts with a path ending with a slash, so the slash is not necessary. But this means subsequent iterations will need a "/" appended to the end.

It might be more clear if we make the naming convention more explicit.

For example:
- function copyDirectory($sourceDir, $destinationDir)  -- and document the function to explain it needs a trailing slash
- $newPath = $sourceDir . $name;
- if (is_dir($newPath)) { copyDirectory($newPath...

The function would still work even with extra //'s thanks to how the OS handles it, however making these changes would eliminate any ambiguity and avoid the "not sure".

Reply:
This was put here for testing purposes, our local environment is not setup to easily test this right now so I put the comment here to make debugging easier once we got to staging.